services/QuillLMS/spec/models/concept_feedback_spec.rb

```diff
@@ -84,4 +74,25 @@
     end
   end
 
+  describe '#translations_json(language:)' do
+    subject { concept_feedback.translations_json(locale: )}
```

Comment:
Style nitpick: I think you want to move the space inside the final parens to outside the parens and inside the curly brace.  Interesting that we don't have a linting rule for consistent spacing like this.

Reply:
I will make a rubocop for this and put it in another PR